Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml

```
@@ -15,6 +15,7 @@
     ClothingOuterWinterMiner: 2
     ClothingNeckScarfStripedBrown: 3
     ClothingShoesBootsWinterCargo: 2
+    MailBag: 1
```

```suggestion
    MailBag: 2
```
I think we can add another one so there is no rush to the vendomat at round start